src/include/nodes/parsenodes.h

```diff
@@ -1753,6 +1753,14 @@ typedef struct SelectStmt
 	struct SelectStmt *larg;	/* left child */
 	struct SelectStmt *rarg;	/* right child */
 	/* Eventually add fields for CORRESPONDING spec here */
+
+	/* These fields are used only in SelectStmt with PIVOT keyword */
+	bool	isPivot;
+	struct SelectStmt *srcSql;
+	struct SelectStmt *catSql;
+	List	*value_col_strlist;
+	char	*pivotCol;
+	Node	*aggFunc;
```

Comment:
Since you are adding in new fields to `SelectStmt` please refer to the following fix by @timchang514 where he fixed a problem with uninitialized value when copying these types of structs.

```
commit 26b4e995d4b69e934c511019ee2175a31227c56a
Author: Tim Chang <timchang514@users.noreply.github.com>
Date:   Mon Sep 25 15:14:17 2023 -0400

    Add new fields to IntoClause funcs to avoid issues with uninitialized memory. (#219)

    Signed-off-by: Tim Chang <changti@amazon.com>
```

Also, I found your branch based on a really old commit. You may need to rebase.